server/src/main/java/org/apache/druid/metadata/SQLMetadataSupervisorManager.java

```diff
@@ -91,24 +89,19 @@ public void start()
   public void insert(final String id, final SupervisorSpec spec)
   {
     dbi.withHandle(
-        new HandleCallback<Void>()
-        {
-          @Override
-          public Void withHandle(Handle handle) throws Exception
-          {
-            handle.createStatement(
-                StringUtils.format(
-                    "INSERT INTO %s (spec_id, created_date, payload) VALUES (:spec_id, :created_date, :payload)",
-                    getSupervisorsTable()
-                )
-            )
-                  .bind("spec_id", id)
-                  .bind("created_date", DateTimes.nowUtc().toString())
-                  .bind("payload", jsonMapper.writeValueAsBytes(spec))
-                  .execute();
+        (HandleCallback<Void>) handle -> {
+          handle.createStatement(
+              StringUtils.format(
+                  "INSERT INTO %s (spec_id, created_date, payload) VALUES (:spec_id, :created_date, :payload)",
+                  getSupervisorsTable()
+              )
+          )
+                .bind("spec_id", id)
+                .bind("created_date", DateTimes.nowUtc().toString())
+                .bind("payload", jsonMapper.writeValueAsBytes(spec))
+                .execute();
 
-            return null;
-          }
+          return null;
```

Comment:
This can be removed and the lambda can then be shortened by removing the braces `{ ... }`.